src/network_download_file.cpp

```diff
@@ -53,7 +53,11 @@ namespace network
             curl_easy_setopt(curl, CURLOPT_FRESH_CONNECT, 1L);
             curl_easy_setopt(curl, CURLOPT_FAILONERROR, 1L);
             curl_easy_setopt(curl, CURLOPT_TIMEOUT_MS, 5000L);
-            curl_easy_setopt(curl, CURLOPT_PROTOCOLS, CURLPROTO_HTTPS);
+            #if LIBCURL_VERSION_NUM >= 0x075500
```

Comment:
Is that the right number? We need 7.85 or later, and that looks like 7.55.

Reply:
`LIBCURL_VERSION_NUM` is a hexadecimal number, see https://github.com/curl/curl/blob/b9c2a56be2980c1d2f6192006ac5d306bbb6858b/include/curl/curlver.h#L54.

Reply:
OK, sorry for raising it.